contracts/Authorizable.sol

```diff
@@ -4,18 +4,24 @@ pragma solidity >=0.4.22 <0.9.0;
 import "@openzeppelin/contracts/access/Ownable.sol";
 
 contract Authorizable is Ownable {
-    address public trustee;
+    mapping(address => bool) private trustees;
```

Comment:
Trustee can be public. Otherwise, you need to find trustee from the historical transaction. It is relative inconvenient.